src/theme/dark/colors.js

```diff
@@ -56,6 +56,8 @@ const appColors = {
   nodeBadgeBackground: rawColors.neutral.limedSpruce,
   nodeBadgeBorder: rawColors.neutral.bluebayoux,
   nodeBadgeColor: rawColors.neutral.white,
+  //Table
+  tableRowBg: rawColors.neutral.shark,
```

Comment:
I believe we should not use any new colors here, so would it be possible to sync with the design team and use one of the existing ones?

Reply:
This has been added by design team, @christophidesp